<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_tel</class>
 <widget class="QDialog" name="add_tel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>add telephone</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>telephone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_tel"/>
     </item>
     <item>
      <widget class="QComboBox" name="cb_type"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>243</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="tb_ok">
     <property name="text">
      <string>...</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="a_add_tel">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/pic/tick.png</normaloff>:/new/prefix1/pic/tick.png</iconset>
   </property>
   <property name="text">
    <string>add tel</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
